starter-code/css/main.css

```diff
@@ -8,3 +8,86 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+.blue-background{
+  height: 250px;
+  background: -webkit-linear-gradient(#1c5380, #06365f);
+}
+.phones{
+  float:left;
+  position: relative;
+  top: -180px;
+  padding-left: 80px;
+}
+.brand{
+  float: left;
+  position:relative;
+  top: -160px;
+  padding-left: 50px;
+}
+.login{
+  float: left;
+  position: relative;
+  top: -145px;
+  left: 85px;
+  padding: 5px 10px 8px 10px;
+  background: -webkit-linear-gradient(#1c5380, #06365f);
+  border: ridge 0.4px #06365f;
+  font-weight: bold;
+  font-size: 14px;
+  color: white;
+  float: left;
+}
+.home{
+  position: relative;
+  top: 5px;
+}
+.frame{
+  float:left;
+  position: relative;
+  top: -105px;
+  right: 318px;
+}
+.text{
+  float:left;
+  width: 400px;
+  position: relative;
+  top: -380px;
```

Comment:
Try to avoid using such large values when positioning your elements on a page, especially negative ones. It can have a big impact on how your page looks in multiple resolutions and devices. There are usually better ways to align your elements the way you want them to. If you try viewing your page in multiple widths, you'll see what I mean.